test/client/components/BrowseTopicsModal.test.js

```diff
@@ -59,42 +58,28 @@ describe('BrowseTopicsModal', () => {
     }
   ]
 
-  const initialState = {
-    communities: {
-      coo: community
-    },
-    currentCommunityId: '1',
-    tagsByQuery: {
-      'subject=community&id=coo': tags
-    },
-    totalTagsByQuery: tags.length
-  }
-
   describe('with onboarding set', () => {
+    var props
+
     beforeEach(() => {
-      currentUser = {
-        memberships: [
-          {...community, community_id: '1'}
-        ]
+      props = {
+        onboarding: true,
+        community,
+        tags
       }
```

Comment:
Are these 3 props the minimum required vales for a render? Make sure to indicate them on the component. If they are not required then consider factoring the test to only begin with what is required for render.